### PR TITLE
Add missing documentation for CMK support on GCP

### PR DESCRIPTION
**Pull request by nkvuong**
## Changes
- Add missing documentation for CMK support on GCP

## Tests

- [ ] ~`make test` run locally~
- [x] relevant change in `docs/` folder
- [ ] ~covered with integration tests in `internal/acceptance`~
- [ ] ~relevant acceptance tests are passing~
- [ ] ~using Go SDK~

### CODE REVIEW

File: docs/resources/mws_customer_managed_keys.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Please follow this [complete runnable example](../guides/aws-workspace.md) with
 
 You must configure this during workspace creation
 
+For AWS
```
Comment (alexott) [inline docs/resources/mws_customer_managed_keys.md:25]: Let's make headers for them?

File: docs/resources/mws_workspaces.md [hunk 2]
```diff
@@ -339,12 +340,14 @@ You can specify a `token` block in the body of the workspace resource, so that T
...
 On AWS, the following arguments could be modified after the workspace is running:
 
-* `network_id` - (Optional, AWS only) `network_id` from [networks](mws_networks.md). Modifying [networks on running workspaces](mws_networks.md#modifying-networks-on-running-workspaces) would require three separate `terraform apply` steps.
-* `credentials_id` - (AWS only) `credentials_id` from [credentials](mws_credentials.md)
-* `storage_customer_managed_key_id` - (Optional, AWS only) `customer_managed_key_id` from [customer managed keys](mws_customer_managed_keys.md) with `use_cases` set to `STORAGE`. This is used to encrypt the DBFS Storage & Cluster EBS Volumes.
-* `private_access_settings_id` - (Optional, AWS only) Canonical unique identifier of [databricks_mws_private_access_settings](mws_private_access_settings.md) in Databricks Account
+* `network_id` - Modifying [networks on running workspaces](mws_networks.md#modifying-networks-on-running-workspaces) would require three separate `terraform apply` steps.
+* `credentials_id`
+* `storage_customer_managed_key_id`
+* `private_access_settings_id`
```
Comment (alexott) [inline docs/resources/mws_workspaces.md:350]: Too much was removed?

**Review by alexott — CHANGES_REQUESTED** [review]

File: docs/resources/mws_customer_managed_keys.md (hunk 1 above)
Comment (nkvuong) [reply docs/resources/mws_customer_managed_keys.md]: done

File: docs/resources/mws_workspaces.md (hunk 2 above)
Comment (nkvuong) [reply docs/resources/mws_workspaces.md:350]: these already have descriptions above - this is just to highlight which parameter can be updated

**Review by alexott — APPROVED** [review]
lgtm

**Review by tanmay-db — APPROVED** [review]

**Pull request merged**